persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/configuration/TableManipulationConfigurationBuilder.java

```diff
@@ -37,11 +36,11 @@
    }
 
    /**
-    * Repetitive DB operations this are batched according to this parameter. This is an optional parameter, and if it
-    * is not specified it will be defaulted to {@link TableManager#DEFAULT_BATCH_SIZE}.
+    * Deprecated. Please use {@link org.infinispan.configuration.cache.AbstractStoreConfigurationBuilder#maxBatchSize(int)} instead.
     */
+   @Deprecated
    public S batchSize(int batchSize) {
-      attributes.attribute(BATCH_SIZE).set(batchSize);
+      attributes.attribute(AbstractJdbcStoreConfiguration.MAX_BATCH_SIZE).set(batchSize);
```

Comment:
Well you can call it on the parent one :) That is why I was saying we would have to change create method, being that it would take another argument.

The thing I am worrying about is that with this change a user having a custom cache store may not be using the batch size properly as it should now be configured on the store level, but their store is most likely using the TableManipulation one. I added another comment that should help alleviate more of my concern.